CHANGELOG.md

```diff
@@ -1,8 +1,18 @@
 # Change Log
 All notable changes to the "azure-cosmosdb" extension will be documented in this file.
+
+## 0.3.0 - 2017-12-15
+### Fixed
```

Comment:
I prefer the 'Added' section above 'Fixed' (it's the more exciting stuff)

Reply:
Fixed Fixed. :-)